index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
 implementing the rule above. This can ensure proper term identification,
 typing, and order, when a JSON document is processed as JSON-LD.
           </p>
+          <p>
+However, the intention of the issuer is not preserved, unless the <code>@context</code>
+is applied, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>,
+and their exact RDF types, and node and edge structure is understood by the verifier.
+          </p>
 
           <p>
-The rule above guarantees semantic interoperability between JSON and JSON-LD for
-literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD
-processors will use the specific mechanism provided and can verify that all
-terms are correctly specified, JSON-based processors implicitly accept the same
-semantics without performing any JSON-LD transformations, but instead by
+These rules alone do not guarantee semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism.
```

Comment:
I do not understand what you mean

Reply:
I clarified in https://github.com/w3c/vc-data-model/pull/1302#discussion_r1358952470